ext/ffi/lib.rs

```diff
@@ -780,8 +779,16 @@ fn make_sync_fn<'s>(
 
   #[cfg(not(target_os = "windows"))]
   let mut fast_allocations: Option<*mut ()> = None;
+
+  #[cfg(target_os = "linux")]
+  let selinux_is_enabled =
```

Comment:
Presumably this could also be outside the function call entirely?

Reply:
You want me to use thread_local! or lazy_static ?